src/main/java/net/dv8tion/jda/internal/entities/GuildImpl.java

```diff
@@ -1427,75 +1391,81 @@ private AuditableRestAction<Void> timeoutUntilById0(@Nonnull String userId, @Nul
 
     @Nonnull
     @Override
-    public AuditableRestAction<Void> deafen(@Nonnull Member member, boolean deafen)
+    public AuditableRestAction<Void> deafen(@Nonnull UserSnowflake user, boolean deafen)
     {
-        Checks.notNull(member, "Member");
-        checkGuild(member.getGuild(), "Member");
+        Checks.notNull(user, "User");
         checkPermission(Permission.VOICE_DEAF_OTHERS);
```

Comment:
Doesn't look like we do a `checkPosition` for deafening. An oversight?

Reply:
You actually can deafen anyone afaik

Reply:
Even the owner?

Reply:
Yep.